lib/screens/events/eventsCard.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:ecellapp/core/res/colors.dart';
```

Comment:
🔴 
1. file name - `event_card.dart`
2. name of file and name of class in the file should be in sync. You shouldn't call the file `eventsCard` and the class inside, `EventCard`